README.md

```diff
@@ -10,16 +10,25 @@ Follow the steps below to get started with the Node Template, or get it up and r
 your browser in just a few clicks using
 the [Substrate Playground](https://docs.substrate.io/playground/) :hammer_and_wrench:
 
-### Using Nix
+### Repository Set-Up
 
-Install [nix](https://nixos.org/) and optionally [direnv](https://github.com/direnv/direnv) and
-[lorri](https://github.com/target/lorri) for a fully plug and play experience for setting up the
-development environment. To get all the correct dependencies activate direnv `direnv allow` and
-lorri `lorri shell`.
+After cloning this repository, be sure to include necessary submodules.
+
+```sh
+git submodule init
+git submodule update
+```
 
 ### Rust Setup
 
-First, complete the [basic Rust setup instructions](./docs/rust-setup.md).
+If you do not already have Rust installed, with the nightly build set as default, complete the [installation](https://www.rust-lang.org/tools/install).
+
+Following the installation of Rust, ensure your environment is defaulting to the nightly build:
+
+```sh
+rustup default nightly
+rustup target add wasm32-unknown-unknown --toolchain nightly
+```
```

Comment:
we could collapse the repo & rust setup into a single section called: Setup

add information about pre-reqs (dev dependencies) such as git (should be obvious, but worth being exhaustive), rustup, and other dependencies devs will need to run setup scripts (or, eventually, all the commands we put into `setup.sh`

Reply:
My only hesitation here is that the submodule setup is always necessary. Setting up Rust isn't if you're using Docker exclusively. We should consider adding arguments to the setup script to carry out the relevant operations depending on which type of environment you want. So `./setup.sh docker`, `./setup.sh build`, something like that.

Reply:
Yeah, "setup" could have a few paths. One, if you want to run node locally, two for Docker, and so forth. As a developer, I don't want a wall of text when I start something -- just give me the bottom line. We want to think about the barrier of entry for developers and the curious, and the lower we can maintain that barrier, the better.

Reply:
Agreed. We're on a good track right now. I do think we need an elevator pitch in the README.

Reply:
What problem does this solve? Why is this valuable? Why should you care? What should you do about it?

Can we answer that in 3-5 sentences?

Reply:
We've got answers to those questions, but we are in the process of cleaning the language up.